### PR TITLE
Code Review Updates

### PR DESCRIPTION
**Pull request by otienoelvis**
Added Support Page Module
Moved IFRAME links to config files
Remove hardcoded password strings
Added Linting, Prevents commits with "Console.log"
Updated Readme

### CODE REVIEW

**Review by Kambreezy — APPROVED** [review]
lgtm

File: src/app/layouts/full/header/header.component.html
Comment (gmm2204) [inline src/app/layouts/full/header/header.component.html:1]: Confirm this link's binding

Comment (otienoelvis) [reply src/app/layouts/full/header/header.component.html:1]: fixed on https://github.com/CENTERS-FOR-INTERNATIONAL-PROGRAMS/dmi-shield-frontend/pull/69/commits/f2a6be9d76b7a321a0b818cc6d112f33be764744

**Review by Kambreezy — APPROVED** [review]
lgtm

**Pull request merged**